alvr/server/cpp/alvr_server/ClientConnection.cpp

```diff
@@ -263,6 +267,81 @@ void ClientConnection::ProcessRecv(unsigned char *buf, size_t len) {
 			OnFecFailure();
 		}
 	}
+	FillStatistics();
+}
+
+void ClientConnection::FillStatistics() {
+	uint64_t now = GetTimestampUs();
+	if (now - m_LastStatisticsFillerUpdate > STATISTICS_FILLER_US && now - m_LastStatisticsUpdate > STATISTICS_FILLER_US && now - m_LastStatisticsUpdate < STATISTICS_TIMEOUT_US)
+	{
+		//timings might be a little incorrect since it is a mix from a previous sent frame and latest frame
```

Comment:
Do we really want to show any timing info then? Providing bad data is arguably worse than no data until the client is reconnected.

Reply:
I did not change the comment, that should be for rendered frames
Frames are still rendered while the client connection is broken or lagging (server fps reflects this)
I can remove it if the timings should only matter for frames sent to the client, though

Reply:
Resolving as it's better to keep this PR isolated to one thing.